pyrolite/geochem/alteration.py

```diff
@@ -127,3 +127,35 @@ def WIP(df: pd.DataFrame):
 
     """
     return 2 * df.Na2O / 0.35 + df.MgO / 0.9 + 2 * df.K2O / 0.25 + df.CaO / 0.7
+
+@update_docstring_references
+def AI(df: pd.DataFrame):
```

Comment:
Might want to rename this one; having a function called `AI` might be a bit loaded! Perhaps `IAI` or even `IshikawaAltIndex` if you'd like to be explicit.